barretenberg/cpp/src/barretenberg/protogalaxy/combiner_example_gen.py

```diff
@@ -3,7 +3,7 @@
 
 # np.set_printoptions(formatter={'int': hex})
 
-EXTENDED_RELATION_LENGTH = 13
+EXTENDED_RELATION_LENGTH = 12
```

Comment:
interesting that this decreased

Reply:
A prior line of work decreased it (using different etas rather than powers of eta)  but because nothing in the logic changed, that PR could just remove one value in the combiner tests and this file wasnt modified. Now I had to rerun the script and realised we have one extra eval, but my PR didnt specifically decreased this